mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -6079,12 +6086,24 @@ resource "google_container_cluster" "primary" {
 `, name)
 }
 
+func testAccContainerCluster_basic_1_23_8(name string) string {
+	return fmt.Sprintf(`
+resource "google_container_cluster" "primary" {
+  name               = "%s"
+  location           = "us-central1-a"
+  initial_node_count = 1
+  min_master_version = "1.23.8-gke.1900"
+}
+`, name)
+}
+
 func testAccContainerCluster_withMonitoringConfigEnabled(name string) string {
        return fmt.Sprintf(`
 resource "google_container_cluster" "primary" {
   name               = "%s"
   location           = "us-central1-a"
   initial_node_count = 1
+	min_master_version = "1.23.8-gke.1900"
```

Comment:
The indentation in all the terraform changes seems to use tabs instead of spaces - could you correct this?